### ISSUE TITLE
"moment.tz is not a function" error when using momentTimezonePlugin

### ISSUE THREAD

**Issue by seb-jean** [fullcalendar/fullcalendar#4530]
Hi,
With this code, I have error `moment.tz is not a function` :

```javascript
import { Calendar } from '@fullcalendar/core';
import { toMoment } from '@fullcalendar/moment'; // only for formatting
import momentTimezonePlugin from '@fullcalendar/moment-timezone';

let calendar = new Calendar(calendarEl, {
  plugins: [ momentTimezonePlugin ],
  timeZone: 'America/New_York',
  events: [
    { start: '2018-09-01T12:30:00Z' }, // will be shifted to America/New_York
    { start: '2018-09-01T12:30:00+XX:XX' }, // will be shifted to America/New_York
    { start: '2018-09-01T12:30:00' } // will be parsed as America/New_York
  ],
  dateClick: function(arg) {
    // millisecond value is correctly in America/New_York
    console.log(arg.date.valueOf())

    // use the plugin for manipulation and formatting
    let m = toMoment(calendar, arg.date)
    console.log(m.format()) // something like '2018-09-01T12:30:00-05:00'
  }
});
```
If I add this follow code, it's work:

```javascript
import moment from 'moment';
import 'moment-timezone';
```

**Comment by acerix** [comment]
Thanks for the report and workaround

**Comment by arshaw** [comment]
fixed in master 9a647f0f81562abbbcb57aec65ca6714599f3943

**Comment by arshaw** [comment]
[V4 was officially released a few days ago!](https://fullcalendar.io/blog/2019/03/v4-officially-released)
